src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/jira/StandaloneStoppedNode.kt

```diff
@@ -153,6 +153,13 @@ internal data class StandaloneStoppedNode(
                 break
             }
             if (deadline < now()) {
+                if (logger.isDebugEnabled) {
+                    val logContent = ssh.safeExecute(
+                        cmd = "cat $jiraHome/log/atlassian-jira.log",
+                        timeout = ofMinutes(3)
+                    ).output
+                    logger.debug("Jira log file: <<$logContent>>")
```

Comment:
I was thinking about TRACE level of log, however for our internal test usage it doesn't matter and I'm not really sure which one is proper. Do you think DEBUG is fine?

Reply:
I think it's fine